packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -399,24 +402,28 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 				/>
 			) }
 			<div className={ 'design-picker__grid' }>
-				{ filteredDesigns.map( ( design ) => (
-					<DesignButtonContainer
-						key={ design.slug }
-						design={ design }
-						locale={ locale }
-						onSelect={ onSelect }
-						onPreview={ onPreview }
-						onUpgrade={ onUpgrade }
-						highRes={ false }
-						hideFullScreenPreview={ false }
-						hideDesignTitle={ false }
-						isPremiumThemeAvailable={ isPremiumThemeAvailable }
-						previewOnly={ previewOnly }
-						hasDesignOptionHeader={ hasDesignOptionHeader }
-						onCheckout={ onCheckout }
-						verticalId={ verticalId }
-						hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
-					/>
+				{ filteredDesigns.map( ( design, index ) => (
+					<Fragment key={ design.slug }>
+						<DesignButtonContainer
+							design={ design }
+							locale={ locale }
+							onSelect={ onSelect }
+							onPreview={ onPreview }
+							onUpgrade={ onUpgrade }
+							highRes={ false }
+							hideFullScreenPreview={ false }
+							hideDesignTitle={ false }
+							isPremiumThemeAvailable={ isPremiumThemeAvailable }
+							previewOnly={ previewOnly }
+							hasDesignOptionHeader={ hasDesignOptionHeader }
+							onCheckout={ onCheckout }
+							verticalId={ verticalId }
+							hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
+						/>
+						{ ( ( filteredDesigns.length < 3 && index === filteredDesigns.length - 1 ) ||
```

Comment:
Is it possible to insert the blank canvas design into `filteredDesigns`?

Reply:
It's good idea but I need some time for it and https://github.com/Automattic/wp-calypso/pull/66895#discussion_r954482596

Reply:
It's good idea but I need some time for it and https://github.com/Automattic/wp-calypso/pull/66895#discussion_r954482596

Reply:
@arthur791004 can a blank canvas design be expected from the API, as I can see the \`isBlankCanvasDesign\` is used [here](https://github.com/Automattic/wp-calypso/blob/22f0182bc9427fda885cf17e3781dc18ee220754/packages/design-picker/src/components/unified-design-picker.tsx#L304).

Reply:
@arthur791004 I created a blank canvas design and insert to the static designs list, could you have a look at the implementation? Thanks